src/Schema/Types/LaravelEnumType.php

```diff
@@ -75,13 +79,22 @@ public static function classMustExtendBenSampoEnumEnum(string $enumClass): \Inva
         return new \InvalidArgumentException("Class {$enumClass} must extend {$baseClass}.");
     }
 
+    /**
+     * @template LocalTValue
+     * @template LocalTEnum of \BenSampo\Enum\Enum<LocalTValue>
+     *
+     * @param LocalTEnum $value
```

Comment:
no type variables necessary

```suggestion
     * @param \BenSampo\Enum\Enum<mixed> $value
```